lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -50,10 +50,11 @@ cloudbridge==3.1.0 ; python_version >= "3.7" and python_version < "3.12"
 colorama==0.4.6 ; python_version >= "3.7" and python_version < "3.12" and platform_system == "Windows"
 coloredlogs==15.0.1 ; python_version >= "3.7" and python_version < "3.12"
 conda-package-streaming==0.7.0 ; python_version >= "3.7" and python_version < "3.12"
-cryptography==40.0.1 ; python_version >= "3.7" and python_version < "3.12"
+cryptography==40.0.2 ; python_version >= "3.7" and python_version < "3.12"
 cwl-upgrader==1.2.4 ; python_version >= "3.7" and python_version < "3.12"
-cwl-utils==0.23 ; python_version >= "3.7" and python_version < "3.12"
+cwl-utils==0.24 ; python_version >= "3.7" and python_version < "3.12"
 cwltool==3.1.20221109155812 ; python_version >= "3.7" and python_version < "3.12"
+cython==0.29.34 ; python_version >= "3.7" and python_version < "3.12"
```

Comment:
I think this is coming from pysam 0.21.0, which started using pyproject.toml , but shouldn't be needed.